clients/lsp-php.el

```diff
@@ -85,6 +85,12 @@
   :link '(url-link "https://github.com/bmewburn/vscode-intelephense")
   :package-version '(lsp-mode . "6.1"))
 
+(defcustom lsp-intelephense-php-version "8.0"
+  "Minimum version of PHP to refer to. Affects code actions, diagnostic & completions."
+  :type 'string
+  :group 'lsp-mode
+  :package-version '(lsp-mode . "6.1"))
```

Comment:
please change it to 8.0.1

Reply:
Thanks, I wasn't sure what to set it to :)